turbopack/crates/turbopack-ecmascript-runtime/js/src/browser/dev/runtime/dom/runtime-backend-dom.ts

```diff
@@ -236,53 +236,61 @@ async function loadWebAssemblyModule(
     const chunkUrl = getChunkRelativeUrl(chunkPath);
     const decodedChunkUrl = decodeURI(chunkUrl);
 
-    if (chunkPath.endsWith(".css")) {
-      const previousLinks = document.querySelectorAll(
-        `link[rel=stylesheet][href="${chunkUrl}"],link[rel=stylesheet][href^="${chunkUrl}?"],link[rel=stylesheet][href="${decodedChunkUrl}"],link[rel=stylesheet][href^="${decodedChunkUrl}?"]`
-      );
-      if (previousLinks.length > 0) {
-        // CSS chunks do not register themselves, and as such must be marked as
-        // loaded instantly.
-        resolver.resolve();
+    if (typeof importScripts === "function") {
+      if (chunkPath.endsWith(".js")) {
+        importScripts(TURBOPACK_WORKER_LOCATION + chunkUrl);
```

Comment:
Why do we need a global for this when this is always `location.origin`?

Also I feel like this need to have `CHUNK_BASE_PATH` too. Do we have a test case using `import("...")` in a WebWorker?